ekncontent/ekncontent/eknc-query-object-private.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2018 Endless Mobile, Inc. */
+
+#pragma once
+
+#include <glib.h>
+#include <xapian-glib.h>
+
```

Comment:
You should `#include "eknc-query-object.h"` here, so you can drop the include of the public header

Reply:
Fixed.